conconartist/lyric-lava
# Feature/form refactor

#### Purpose: 
> Check all that apply.
### What does this PR do? 
> Clearly and concisely describe changes made. 

Complete functionality for MVP including error handling, loading, and React Router.  UX is improved with list handling for results-- moving to a new page if the list is over 10 results.  All buttons have functionality and state should be properly updated.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

App.js is updated.  Error.js, Load.js, and WordList.js are all newly updated and modified.  Some CSS was updated to clean up the layout and test UX.

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Run `npm start` in terminal when getting to the project directory.  Open browser window and go to the url http://localhost:3000 to interact with the app.  Enter words into the search bar to see results and click on the prompt generator for prompts.

### Steps To Take
> Any questions and possible steps moving forward.

Next will be E2E testing with further refactoring and adding nice to have feature (favoriting prompts).

#### Checklist:
